apps/trading/lib/hooks/use-navigate-to-last-market.spec.ts

```diff
@@ -0,0 +1,60 @@
+import * as router from 'react-router';
+import { useNavigateToLastMarket } from './use-navigate-to-last-market';
+import { useGlobalStore } from '../../stores';
+import { renderHook } from '@testing-library/react';
+import { useTopTradedMarkets } from './use-top-traded-markets';
+import { Links } from '../links';
+
+const mockLastMarketId = 'LAST';
+
+jest.mock('../../stores', () => {
+  const original = jest.requireActual('../../stores');
+  return {
+    ...original,
+    useGlobalStore: jest.fn(),
+  };
+});
+
+jest.mock('./use-top-traded-markets', () => {
+  return {
+    useTopTradedMarkets: jest.fn(),
+  };
+});
+
+describe('useNavigateToLastMarket', () => {
+  const navigate = jest.fn();
+  beforeAll(() => {
+    jest.spyOn(router, 'useNavigate').mockImplementation(() => navigate);
```

Comment:
I think its cleaner/simpler to just wrap the rendering of the hook with a memory router

```
const setup = () => {
  const wrapper = ({ children }: { children: ReactNode }) => (
    <MemoryRouter>{children}</MemoryRouter>
  );
  return renderHook(() => useNavigateToLastMarket(), { wrapper });
}
```